2019-06-19-all-hands-meeting.md

```diff
@@ -0,0 +1,113 @@
+    - [Governance][gov-wg]
+        - dc: lots of next steps, but in a bit of holding. possible mid-july ramp up.
+    - [Infra & Ops][ino-wg]
+        - [when2meet posted](https://www.when2meet.com/?7903449-KEcbc) by @elon.
+- Discussion
+    - Aether project checkin (ben)
+        - intro call: ben, yurko, pat + other interested
+        - [ ] #action next all-hands meeting discsusion of contracts.
+        - [ ] #action gather contract examples (all)
+    - Revisit hackfoldr for meeting index
+    - Task tracker update (patcon) (patcon)
+    - Coworking? (patcon)
+    - Infrastructure call agenda (ben)
+
+## Announcements
+
+- Aether project checkin
+    - research previous examples of work arrangements
+        - external hourly rate? or other ways
+        - patcon constant struggle by hourly rate
+        - dawn prefers by hour
+        - yurko in the end it's by hour anyways (hours are a commodity)
+        - client perspective vs. project team perspective vs. cooperative perspective
+
+## Notes
+
+- aether
+    - first call: what is aether. hellos.
+    - second call: business. engagement. #todo convo about contracts
+        - scope of work (in proposal hackmd)
+    - july call
+        - logisitics. schedules.
+        - dcwalk: wanting to collect examples of pricing/contract before 2nd meeting.
+    - Should have meeting about this before 2nd aether meeting
+- short link service
+    - discussion about current workflow
+        - hackpad folder vs links
+    - Question about moving forawrd with shrot link service
+        - Yurko said it may be over enginered but its functional so no block
+        - Elon said it was fine the way it was
+        - No one blocked the use of the service
+    - moving forward with incorporating shot link service into workflow
+    - ben talked about how infrustucture group should have say in tools we use and take leadership driving that
+        - In business you are told what tools to use
+        - You are given the tools to use them
+        - We should entrust the infra WG to take on that decision
+    - ben - Some other services have not been fully adopted
+        - Time Tracking - we need to start putting time in
+        - Password Manager
+        - Better to "complete" (e.g. Loomio) one thing than start many things (e.g. shortlink, hackfoldr, password manager, timetracker are in this state atm)
+        - Completion means pushing one thing into our regular workflow, whereas things that are spun-up but not adopted by all becomes a burden to maintain and adds to cog overload
+
+## Follow Up
+
+### Actionable Items
+
+- [ ] patcon: Review past #todos and recover potentials lost items
+- [ ] Consensus by team members and cooperative about how to split $
+    - [ ] All: Research ahead of next meeting
+    - [ ] All: Discuss this item at next all-hands
+- [ ] Ben wants to on our next call try equal speaking time
```

Comment:
```suggestion
```

Reply:
I don't see how this works as a todo, it might be better as a point/proposal in notes above? Or an "upcoming discussion"?

Reply:
We discussed this and people don't seem reluctant to try. Since I cannot be at the meeting, perhaps next meeting's `Cultivator` can revisit at start of meeting to see if people want do try this?

>- [ ] Cultivator: Poll if people would like to go through this meeting with more equal speaking time